2024-10-06 20:47:47,313 INFO    StreamThr :7111 [internal.py:wandb_internal():85] W&B internal server running at pid: 7111, started at: 2024-10-06 20:47:47.312458
2024-10-06 20:47:47,314 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: status
2024-10-06 20:47:47,315 INFO    WriterThread:7111 [datastore.py:open_for_write():87] open: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/run-inycqnme.wandb
2024-10-06 20:47:47,317 DEBUG   SenderThread:7111 [sender.py:send():379] send: header
2024-10-06 20:47:47,317 DEBUG   SenderThread:7111 [sender.py:send():379] send: run
2024-10-06 20:47:47,551 INFO    SenderThread:7111 [dir_watcher.py:__init__():211] watching files in: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files
2024-10-06 20:47:47,551 INFO    SenderThread:7111 [sender.py:_start_run_threads():1188] run started: inycqnme with start time 1728265667.312975
2024-10-06 20:47:47,556 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: check_version
2024-10-06 20:47:47,556 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: check_version
2024-10-06 20:47:47,642 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: run_start
2024-10-06 20:47:47,650 DEBUG   HandlerThread:7111 [system_info.py:__init__():26] System info init
2024-10-06 20:47:47,651 DEBUG   HandlerThread:7111 [system_info.py:__init__():41] System info init done
2024-10-06 20:47:47,651 INFO    HandlerThread:7111 [system_monitor.py:start():194] Starting system monitor
2024-10-06 20:47:47,651 INFO    SystemMonitor:7111 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-10-06 20:47:47,651 INFO    HandlerThread:7111 [system_monitor.py:probe():214] Collecting system info
2024-10-06 20:47:47,653 INFO    SystemMonitor:7111 [interfaces.py:start():188] Started cpu monitoring
2024-10-06 20:47:47,654 INFO    SystemMonitor:7111 [interfaces.py:start():188] Started disk monitoring
2024-10-06 20:47:47,655 ERROR   HandlerThread:7111 [gpu_apple.py:probe():176] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 164, in probe
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 20:47:47,657 INFO    SystemMonitor:7111 [interfaces.py:start():188] Started gpuapple monitoring
2024-10-06 20:47:47,658 ERROR   gpuapple  :7111 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 20:47:47,660 DEBUG   HandlerThread:7111 [system_info.py:probe():152] Probing system
2024-10-06 20:47:47,660 INFO    SystemMonitor:7111 [interfaces.py:start():188] Started memory monitoring
2024-10-06 20:47:47,661 INFO    SystemMonitor:7111 [interfaces.py:start():188] Started network monitoring
2024-10-06 20:47:47,663 DEBUG   HandlerThread:7111 [gitlib.py:_init_repo():56] git repository is invalid
2024-10-06 20:47:47,664 DEBUG   HandlerThread:7111 [system_info.py:probe():200] Probing system done
2024-10-06 20:47:47,664 DEBUG   HandlerThread:7111 [system_monitor.py:probe():223] {'os': 'macOS-14.6.1-arm64-arm-64bit', 'python': '3.12.2', 'heartbeatAt': '2024-10-07T01:47:47.661090', 'startedAt': '2024-10-07T01:47:47.304074', 'docker': None, 'cuda': None, 'args': ('data.pkl',), 'state': 'running', 'program': '/Users/milind/Desktop/valence/milind_code/non_llm.py', 'codePathLocal': 'non_llm.py', 'codePath': 'non_llm.py', 'host': 'Milinds-MacBook-Pro.local', 'username': 'milind', 'executable': '/opt/anaconda3/bin/python', 'cpu_count': 8, 'cpu_count_logical': 8, 'disk': {'/': {'total': 228.27386474609375, 'used': 165.25278091430664}}, 'gpuapple': {'type': 'arm', 'vendor': 'Apple'}, 'memory': {'total': 8.0}}
2024-10-06 20:47:47,664 INFO    HandlerThread:7111 [system_monitor.py:probe():224] Finished collecting system info
2024-10-06 20:47:47,664 INFO    HandlerThread:7111 [system_monitor.py:probe():227] Publishing system info
2024-10-06 20:47:47,664 DEBUG   HandlerThread:7111 [system_info.py:_save_conda():209] Saving list of conda packages installed into the current environment
2024-10-06 20:47:48,557 INFO    Thread-12 :7111 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files/conda-environment.yaml
2024-10-06 20:47:49,675 ERROR   gpuapple  :7111 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 20:47:49,801 DEBUG   HandlerThread:7111 [system_info.py:_save_conda():224] Saving conda packages done
2024-10-06 20:47:49,802 INFO    HandlerThread:7111 [system_monitor.py:probe():229] Finished publishing system info
2024-10-06 20:47:49,805 DEBUG   SenderThread:7111 [sender.py:send():379] send: files
2024-10-06 20:47:49,805 INFO    SenderThread:7111 [sender.py:_save_file():1454] saving file wandb-metadata.json with policy now
2024-10-06 20:47:50,049 INFO    wandb-upload_0:7111 [upload_job.py:push():130] Uploaded file /var/folders/jv/wvjkll0d0vl_clbchr37hpk40000gn/T/tmpdmqyd7q4wandb/n5fi9ch3-wandb-metadata.json
2024-10-06 20:47:50,089 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: python_packages
2024-10-06 20:47:50,090 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: python_packages
2024-10-06 20:47:50,090 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: stop_status
2024-10-06 20:47:50,090 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: stop_status
2024-10-06 20:47:50,091 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 20:47:50,257 DEBUG   SenderThread:7111 [sender.py:send():379] send: telemetry
2024-10-06 20:47:50,567 INFO    Thread-12 :7111 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files/conda-environment.yaml
2024-10-06 20:47:50,567 INFO    Thread-12 :7111 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files/wandb-metadata.json
2024-10-06 20:47:50,568 INFO    Thread-12 :7111 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files/requirements.txt
2024-10-06 20:47:51,095 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 20:47:51,686 ERROR   gpuapple  :7111 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 20:47:52,106 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 20:47:53,101 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 20:47:53,271 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: status_report
2024-10-06 20:47:53,698 ERROR   gpuapple  :7111 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 20:47:54,104 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 20:47:55,109 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 20:47:55,749 ERROR   gpuapple  :7111 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 20:47:56,114 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 20:47:57,119 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 20:47:57,753 ERROR   gpuapple  :7111 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 20:47:58,133 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 20:47:58,682 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 20:47:58,683 DEBUG   SenderThread:7111 [sender.py:send():379] send: history
2024-10-06 20:47:58,683 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: status_report
2024-10-06 20:47:58,683 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: summary_record
2024-10-06 20:47:58,684 INFO    SenderThread:7111 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 20:47:58,689 DEBUG   SenderThread:7111 [sender.py:send():379] send: exit
2024-10-06 20:47:58,689 INFO    SenderThread:7111 [sender.py:send_exit():586] handling exit code: 0
2024-10-06 20:47:58,690 INFO    SenderThread:7111 [sender.py:send_exit():588] handling runtime: 11
2024-10-06 20:47:58,690 INFO    SenderThread:7111 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 20:47:58,690 INFO    SenderThread:7111 [sender.py:send_exit():594] send defer
2024-10-06 20:47:58,690 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: defer
2024-10-06 20:47:58,690 INFO    HandlerThread:7111 [handler.py:handle_request_defer():184] handle defer: 0
2024-10-06 20:47:58,690 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: defer
2024-10-06 20:47:58,690 INFO    SenderThread:7111 [sender.py:send_request_defer():610] handle sender defer: 0
2024-10-06 20:47:58,690 INFO    SenderThread:7111 [sender.py:transition_state():614] send defer: 1
2024-10-06 20:47:58,690 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: defer
2024-10-06 20:47:58,690 INFO    HandlerThread:7111 [handler.py:handle_request_defer():184] handle defer: 1
2024-10-06 20:47:58,690 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: defer
2024-10-06 20:47:58,690 INFO    SenderThread:7111 [sender.py:send_request_defer():610] handle sender defer: 1
2024-10-06 20:47:58,690 INFO    SenderThread:7111 [sender.py:transition_state():614] send defer: 2
2024-10-06 20:47:58,690 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: defer
2024-10-06 20:47:58,690 INFO    HandlerThread:7111 [handler.py:handle_request_defer():184] handle defer: 2
2024-10-06 20:47:58,690 INFO    HandlerThread:7111 [system_monitor.py:finish():203] Stopping system monitor
2024-10-06 20:47:58,690 DEBUG   SystemMonitor:7111 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-10-06 20:47:58,691 DEBUG   SystemMonitor:7111 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-10-06 20:47:58,691 INFO    HandlerThread:7111 [interfaces.py:finish():200] Joined cpu monitor
2024-10-06 20:47:58,691 DEBUG   SystemMonitor:7111 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-10-06 20:47:58,691 INFO    HandlerThread:7111 [interfaces.py:finish():200] Joined disk monitor
2024-10-06 20:47:58,691 INFO    HandlerThread:7111 [interfaces.py:finish():200] Joined gpuapple monitor
2024-10-06 20:47:58,691 INFO    HandlerThread:7111 [interfaces.py:finish():200] Joined memory monitor
2024-10-06 20:47:58,691 INFO    HandlerThread:7111 [interfaces.py:finish():200] Joined network monitor
2024-10-06 20:47:58,691 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: defer
2024-10-06 20:47:58,691 INFO    SenderThread:7111 [sender.py:send_request_defer():610] handle sender defer: 2
2024-10-06 20:47:58,691 INFO    SenderThread:7111 [sender.py:transition_state():614] send defer: 3
2024-10-06 20:47:58,691 DEBUG   SenderThread:7111 [sender.py:send():379] send: telemetry
2024-10-06 20:47:58,691 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: defer
2024-10-06 20:47:58,691 DEBUG   SenderThread:7111 [sender.py:send():379] send: stats
2024-10-06 20:47:58,691 INFO    HandlerThread:7111 [handler.py:handle_request_defer():184] handle defer: 3
2024-10-06 20:47:58,691 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: defer
2024-10-06 20:47:58,691 INFO    SenderThread:7111 [sender.py:send_request_defer():610] handle sender defer: 3
2024-10-06 20:47:58,691 INFO    SenderThread:7111 [sender.py:transition_state():614] send defer: 4
2024-10-06 20:47:58,691 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: defer
2024-10-06 20:47:58,692 INFO    HandlerThread:7111 [handler.py:handle_request_defer():184] handle defer: 4
2024-10-06 20:47:58,692 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: defer
2024-10-06 20:47:58,692 INFO    SenderThread:7111 [sender.py:send_request_defer():610] handle sender defer: 4
2024-10-06 20:47:58,692 INFO    SenderThread:7111 [sender.py:transition_state():614] send defer: 5
2024-10-06 20:47:58,692 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: defer
2024-10-06 20:47:58,692 INFO    HandlerThread:7111 [handler.py:handle_request_defer():184] handle defer: 5
2024-10-06 20:47:58,692 DEBUG   SenderThread:7111 [sender.py:send():379] send: summary
2024-10-06 20:47:58,692 INFO    SenderThread:7111 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 20:47:58,692 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: defer
2024-10-06 20:47:58,692 INFO    SenderThread:7111 [sender.py:send_request_defer():610] handle sender defer: 5
2024-10-06 20:47:58,692 INFO    SenderThread:7111 [sender.py:transition_state():614] send defer: 6
2024-10-06 20:47:58,692 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: defer
2024-10-06 20:47:58,692 INFO    HandlerThread:7111 [handler.py:handle_request_defer():184] handle defer: 6
2024-10-06 20:47:58,692 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: defer
2024-10-06 20:47:58,692 INFO    SenderThread:7111 [sender.py:send_request_defer():610] handle sender defer: 6
2024-10-06 20:47:58,694 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: status_report
2024-10-06 20:47:58,790 INFO    SenderThread:7111 [sender.py:transition_state():614] send defer: 7
2024-10-06 20:47:58,790 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: defer
2024-10-06 20:47:58,790 INFO    HandlerThread:7111 [handler.py:handle_request_defer():184] handle defer: 7
2024-10-06 20:47:58,790 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: defer
2024-10-06 20:47:58,790 INFO    SenderThread:7111 [sender.py:send_request_defer():610] handle sender defer: 7
2024-10-06 20:47:59,610 INFO    Thread-12 :7111 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files/config.yaml
2024-10-06 20:47:59,611 INFO    Thread-12 :7111 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files/wandb-summary.json
2024-10-06 20:47:59,611 INFO    Thread-12 :7111 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files/output.log
2024-10-06 20:47:59,695 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: poll_exit
2024-10-06 20:48:00,691 INFO    SenderThread:7111 [sender.py:transition_state():614] send defer: 8
2024-10-06 20:48:00,691 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: poll_exit
2024-10-06 20:48:00,712 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: defer
2024-10-06 20:48:00,712 INFO    HandlerThread:7111 [handler.py:handle_request_defer():184] handle defer: 8
2024-10-06 20:48:00,712 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: poll_exit
2024-10-06 20:48:00,713 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: defer
2024-10-06 20:48:00,713 INFO    SenderThread:7111 [sender.py:send_request_defer():610] handle sender defer: 8
2024-10-06 20:48:00,714 INFO    SenderThread:7111 [job_builder.py:build():440] Attempting to build job artifact
2024-10-06 20:48:00,714 INFO    SenderThread:7111 [job_builder.py:_get_source_type():580] no source found
2024-10-06 20:48:00,714 INFO    SenderThread:7111 [sender.py:transition_state():614] send defer: 9
2024-10-06 20:48:00,714 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: poll_exit
2024-10-06 20:48:00,714 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: defer
2024-10-06 20:48:00,715 INFO    HandlerThread:7111 [handler.py:handle_request_defer():184] handle defer: 9
2024-10-06 20:48:00,715 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: defer
2024-10-06 20:48:00,715 INFO    SenderThread:7111 [sender.py:send_request_defer():610] handle sender defer: 9
2024-10-06 20:48:00,715 INFO    SenderThread:7111 [dir_watcher.py:finish():358] shutting down directory watcher
2024-10-06 20:48:01,618 INFO    Thread-12 :7111 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files/output.log
2024-10-06 20:48:01,619 INFO    SenderThread:7111 [dir_watcher.py:finish():388] scan: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files
2024-10-06 20:48:01,620 INFO    SenderThread:7111 [dir_watcher.py:finish():402] scan save: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files/requirements.txt requirements.txt
2024-10-06 20:48:01,620 INFO    SenderThread:7111 [dir_watcher.py:finish():402] scan save: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files/output.log output.log
2024-10-06 20:48:01,622 INFO    SenderThread:7111 [dir_watcher.py:finish():402] scan save: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files/config.yaml config.yaml
2024-10-06 20:48:01,624 INFO    SenderThread:7111 [dir_watcher.py:finish():402] scan save: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files/wandb-summary.json wandb-summary.json
2024-10-06 20:48:01,626 INFO    SenderThread:7111 [dir_watcher.py:finish():402] scan save: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files/conda-environment.yaml conda-environment.yaml
2024-10-06 20:48:01,629 INFO    SenderThread:7111 [dir_watcher.py:finish():402] scan save: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files/wandb-metadata.json wandb-metadata.json
2024-10-06 20:48:01,629 INFO    SenderThread:7111 [sender.py:transition_state():614] send defer: 10
2024-10-06 20:48:01,631 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: defer
2024-10-06 20:48:01,631 INFO    HandlerThread:7111 [handler.py:handle_request_defer():184] handle defer: 10
2024-10-06 20:48:01,631 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: defer
2024-10-06 20:48:01,631 INFO    SenderThread:7111 [sender.py:send_request_defer():610] handle sender defer: 10
2024-10-06 20:48:01,631 INFO    SenderThread:7111 [file_pusher.py:finish():169] shutting down file pusher
2024-10-06 20:48:01,718 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: poll_exit
2024-10-06 20:48:01,718 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: poll_exit
2024-10-06 20:48:01,833 INFO    wandb-upload_1:7111 [upload_job.py:push():130] Uploaded file /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files/output.log
2024-10-06 20:48:01,906 INFO    wandb-upload_4:7111 [upload_job.py:push():130] Uploaded file /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files/conda-environment.yaml
2024-10-06 20:48:01,906 INFO    wandb-upload_2:7111 [upload_job.py:push():130] Uploaded file /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files/config.yaml
2024-10-06 20:48:01,906 INFO    wandb-upload_3:7111 [upload_job.py:push():130] Uploaded file /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files/wandb-summary.json
2024-10-06 20:48:01,906 INFO    wandb-upload_0:7111 [upload_job.py:push():130] Uploaded file /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/files/requirements.txt
2024-10-06 20:48:02,112 INFO    Thread-11 (_thread_body):7111 [sender.py:transition_state():614] send defer: 11
2024-10-06 20:48:02,113 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: defer
2024-10-06 20:48:02,113 INFO    HandlerThread:7111 [handler.py:handle_request_defer():184] handle defer: 11
2024-10-06 20:48:02,114 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: defer
2024-10-06 20:48:02,114 INFO    SenderThread:7111 [sender.py:send_request_defer():610] handle sender defer: 11
2024-10-06 20:48:02,114 INFO    SenderThread:7111 [file_pusher.py:join():175] waiting for file pusher
2024-10-06 20:48:02,114 INFO    SenderThread:7111 [sender.py:transition_state():614] send defer: 12
2024-10-06 20:48:02,115 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: defer
2024-10-06 20:48:02,115 INFO    HandlerThread:7111 [handler.py:handle_request_defer():184] handle defer: 12
2024-10-06 20:48:02,116 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: defer
2024-10-06 20:48:02,116 INFO    SenderThread:7111 [sender.py:send_request_defer():610] handle sender defer: 12
2024-10-06 20:48:02,116 INFO    SenderThread:7111 [file_stream.py:finish():601] file stream finish called
2024-10-06 20:48:02,220 INFO    SenderThread:7111 [file_stream.py:finish():605] file stream finish is done
2024-10-06 20:48:02,220 INFO    SenderThread:7111 [sender.py:transition_state():614] send defer: 13
2024-10-06 20:48:02,220 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: defer
2024-10-06 20:48:02,220 INFO    HandlerThread:7111 [handler.py:handle_request_defer():184] handle defer: 13
2024-10-06 20:48:02,221 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: defer
2024-10-06 20:48:02,221 INFO    SenderThread:7111 [sender.py:send_request_defer():610] handle sender defer: 13
2024-10-06 20:48:02,221 INFO    SenderThread:7111 [sender.py:transition_state():614] send defer: 14
2024-10-06 20:48:02,222 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: defer
2024-10-06 20:48:02,222 DEBUG   SenderThread:7111 [sender.py:send():379] send: final
2024-10-06 20:48:02,222 INFO    HandlerThread:7111 [handler.py:handle_request_defer():184] handle defer: 14
2024-10-06 20:48:02,222 DEBUG   SenderThread:7111 [sender.py:send():379] send: footer
2024-10-06 20:48:02,222 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: defer
2024-10-06 20:48:02,222 INFO    SenderThread:7111 [sender.py:send_request_defer():610] handle sender defer: 14
2024-10-06 20:48:02,222 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: poll_exit
2024-10-06 20:48:02,223 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: poll_exit
2024-10-06 20:48:02,223 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: server_info
2024-10-06 20:48:02,223 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: poll_exit
2024-10-06 20:48:02,223 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: get_summary
2024-10-06 20:48:02,223 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: sampled_history
2024-10-06 20:48:02,223 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: server_info
2024-10-06 20:48:02,224 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 20:48:02,259 DEBUG   SenderThread:7111 [sender.py:send_request():406] send_request: poll_exit
2024-10-06 20:48:02,259 INFO    MainThread:7111 [wandb_run.py:_footer_history_summary_info():3991] rendering history
2024-10-06 20:48:02,260 INFO    MainThread:7111 [wandb_run.py:_footer_history_summary_info():4023] rendering summary
2024-10-06 20:48:02,260 INFO    MainThread:7111 [wandb_run.py:_footer_sync_info():3950] logging synced files
2024-10-06 20:48:02,261 DEBUG   HandlerThread:7111 [handler.py:handle_request():158] handle_request: shutdown
2024-10-06 20:48:02,261 INFO    HandlerThread:7111 [handler.py:finish():882] shutting down handler
2024-10-06 20:48:03,229 INFO    WriterThread:7111 [datastore.py:close():296] close: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_204747-inycqnme/run-inycqnme.wandb
2024-10-06 20:48:03,277 INFO    SenderThread:7111 [sender.py:finish():1608] shutting down sender
2024-10-06 20:48:03,278 INFO    SenderThread:7111 [file_pusher.py:finish():169] shutting down file pusher
2024-10-06 20:48:03,278 INFO    SenderThread:7111 [file_pusher.py:join():175] waiting for file pusher
